test/evp_test.c

```diff
@@ -3948,6 +4002,12 @@ static int parse(EVP_TEST *t)
                 return 0;
             }
             t->reason = take_value(pp);
+        } else if (strcmp(pp->key, "Threads") == 0) {
```

Comment:
Should "Threads" be lowercase like the rest?

Reply:
Nope, the others are ctrl names which are lowercase for other algorithms too. But this one is a new key name for test parameters and these are always with a capital letter at the beginning.

Reply:
Okay, no need to update.